projects/cdk/schematics/utils/get-named-import-references.ts

```diff
@@ -0,0 +1,22 @@
+import {arrayFlat, getImports, Node} from 'ng-morph';
+
+export function getNamedImportReferences(
+    namedImport: string,
+    moduleSpecifier?: string | string[],
+): Node[] {
+    const importDeclarations = getImports('**/**', {
+        namedImports: [namedImport],
+        moduleSpecifier: moduleSpecifier || '**/**',
```

Comment:
Add fallback directly to the attribute so you can just write `moduleSpecifier,` here